### PR TITLE
split out sdm-core

### PR DESCRIPTION
**Pull request by jessitron**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,15 +20,16 @@
   "dependencies": {
     "@atomist/automation-client": "https://r.atomist.com/BJxzuM1V20G",
     "@atomist/automation-client-ext-logzio": "https://r.atomist.com/BJvleYiC20f",
-    "@atomist/sdm": "https://r.atomist.com/B1IW6KqC3Cf",
+    "@atomist/sdm": "0.3.0-nortissej.break-out-api.20180622000216",
```
Comment (ddgenome) [inline package.json]: Why is this depending on a branch? Did we not release the new version of SDM?

File: package.json [hunk 2]
```diff
@@ -20,15 +20,16 @@
...
     "app-root-path": "^2.0.1",
     "json-stringify-safe": "^5.0.1",
     "kubernetes-client": "^5.3.0",
     "lodash": "^4.17.5",
     "logzio-nodejs": "^0.4.10",
     "promise-retry": "^1.1.1",
     "serialize-error": "^2.1.0",
-    "winston-logzio": "^1.0.6"
+    "winston-logzio": "^1.0.6",
+    "@atomist/sdm-core": "0.3.1-20180622172537"
```
Comment (ddgenome) [inline package.json]: Typically these are sorted.

File: package.json (hunk 1 above)
Comment (jessitron) [reply package.json]: oh, yeah, that could be updated. it's in master now.
In fact, I wanted to click release and then update everyone, I'll work on that

File: package.json (hunk 2 above)
Comment (jessitron) [reply package.json]: I figure a linter will do that

**Comment by ddgenome** [comment]
I've updated sdm separately. It looks like sdm-core is not used in k8-automation so there is no need to depend on it.

**Pull request closed without merging**